nav2_rviz_plugins/include/nav2_rviz_plugins/nav2_panel.hpp

```diff
@@ -92,12 +92,12 @@ private Q_SLOTS:
 
   // The NavigateToPose action client
   rclcpp_action::Client<nav2_msgs::action::NavigateToPose>::SharedPtr navigation_action_client_;
-  rclcpp_action::Client<nav2_msgs::action::FollowWaypoints>::SharedPtr
+  rclcpp_action::Client<nav2_msgs::action::follow_waypoints>::SharedPtr
```

Comment:
None of this needs to be changed ,we should keep the action msg name as `FollowWaypoints`, its just the string name of the action servers and action clients using the action of type `FollowWaypoints` needs to change to name `follow_waypoints`!

Reply:
Got it!